tiptapy/__init__.py

```diff
@@ -135,6 +135,12 @@ class OrderedList(BaseContainer):
 
 
 def register_renderer(cls):
```

Comment:
I am still confused, should we create this method as a public function? As the end-user of the library, `def convert_any()` and `def to_html(data: Dict) -> str:` are only the mainly used public functions that I would rely upon. cc: @shon @gauravr